### DISCUSSION TITLE
Preview pcam in a viewport

### DISCUSSION THREAD

**Discussion by electroskeptic** [ramokz/phantom-camera#354] [Ideas]
This is the plugin I've been dreaming of, using it now for my game.
It would be VERY helpful though to be able to see preview of what each pcam is looking at, in the viewport.
Right now the only view I can get is from the main camera. Or is there a way to turn on each pcam preview?
Thanks!

**Comment by electroskeptic** [comment]
This is the game I'm making, a continuation of the old 'Stupid Invaders' game.
Would be great if I viewport could update on selected pcam.
![Screenshot_20240731_160959](https://github.com/user-attachments/assets/78493bf2-d6d8-48bb-9738-4d84597fcfe1)

**Comment by ramokz** [comment]
You can preview other `PCams` from the viewfinder by enabling the `Priority Override` toggle in the panel for the one you want to see.
![image](https://github.com/user-attachments/assets/1e32c39a-7908-4c53-b594-cbf755548970)

A bit more is written about it on the [documentation page](https://phantom-camera.dev/priority#priority_override).

**Reply by electroskeptic** [reply]
Thank you!
